### ISSUE TITLE
MP4Box crashes when dashing cbcs audio content

### ISSUE THREAD

**Issue by vladimir-kazakov** [gpac/gpac#1067]
When dashing cbcs audio content, MP4Box crashes. The log file can be found in https://1drv.ms/f/s!Ah-Kf_FqISsklBLXwzP_OIpwG-R9 (see: /cbcs/dash/dashing-audio.log). Other CENC protection schemes are not affected. Dashing video content is not affected as well. The following commands were used to produce the content.

Applying cbcs protection to the audio content:
```
mp4box.exe -crypt "cbcs\crypt.xml" "audio-128k.mp4" -out "cbcs\audio-128k.mp4"
```

Dashing:
```
mp4box.exe -dash 10000 -rap -bound -bs-switching no -sample-groups-traf -profile dashavc264:onDemand "cbcs\audio-128k.mp4" -out "cbcs\dash\audio.mpd" -logs all@debug -lf "cbcs\dash\dashing-audio.log"
```

**Comment by jeanlf** [comment]
OK the issue was injecting empty saio/saiz/senc with audio in cbcs. This should now be fixed (I also removed injecting seig sample group when not needed)

**Comment by vladimir-kazakov** [comment]
Fixed in GPAC version 0.7.2-DEV-rev538-g897ee93e-master. Closing the issue.
